doc/examples/.gitignore

```diff
@@ -1,9 +0,0 @@
-# Web related
-**/*/lib/generated_plugin_registrant.dart
-
-# Platforms
-**/*/android
-**/*/ios
-**/*/web
-**/*/macos
-**/*/test
```

Comment:
why??

Reply:
did this by mistake thanks for noticing that.

Reply:
I actually thought you might had move it to another gitignore but wanted to be sure :p